R/realtime.service.R

```diff
@@ -8,7 +8,7 @@
 #'@importFrom jsonlite toJSON
 #'@import XML
 #'@examples
-#'serviceEndpoint  <-	'http://olga.er.usgs.gov/data/NACCH/GOM_erosion_hazards_metadata.xml'
+#'serviceEndpoint  <-	system.file('extdata', "GOM_erosion_hazards_metadata.xml", package = 'hazardItems')
```

Comment:
I'm not sure if the function can handle a local file.  This could be travis issues you are seeing.